src/lib.rs

```diff
@@ -71,36 +115,31 @@ unsafe fn __user_exception(cause: arch::ExceptionCause, context: arch::Context)
             println!("0x{:x}", addr);
         }
     }
+    println!(" ");
+    println!(" ");
+    println!(" ");
 
-    println!("");
-    println!("");
-    println!("");
-
-    #[cfg(feature = "colors")]
-    println!("{}", RESET);
+    set_color_code(RED);
```

Comment:
I guess it should be `set_color_code(RESET);` - not a big deal since the next thing we do is `halt`